.github/workflows/api-checks.yml

```diff
@@ -0,0 +1,55 @@
+name: Template API Checks
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - app/**
+      - .github/workflows/api-checks.yml
+  pull_request:
+    paths:
+      - app/**
+      - .github/workflows/api-checks.yml
+
+defaults:
+  run:
+    working-directory: ./app
+
+env:
+  # Some commands in the Makefile check if this is set
+  # and add additional parameters to the commands called
+  CI: true
```

Comment:
i don't believe this is necessary, since [according to github docs, github actions always sets CI to true](https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables)

Reply:
Let me test that